README.md

```diff
@@ -103,6 +103,14 @@ devcontainer --workspace-folder . up
 devcontainer exec --workspace-folder . make proto
 ```
 
+#### Manual ad-hoc use of devcontainer on Linux
+
+First the devcontainer needs to be built and then run to do the make proto:
+```bash
+docker build -f .devcontainer/Dockerfile -t eve-api-builder .
+docker run -v $(pwd):/src -w /src -u $(id -u) eve-api-builder make proto
```

Comment:
Each run of this command creates a container and does not delete it on command exit. What if to add '--rm' as I noted in the comment?

Reply:
Sorry I forgot to push the latest yesterday.